prime-router/src/main/resources/db/migration/V39__add_livd_materialized_view.sql

```diff
@@ -0,0 +1,102 @@
+SELECT
+    r.lookup_table_row_id
+     , r.lookup_table_version_id
+     , r.row_num
+     , r.data->>'Manufacturer'                   as "manufacturer"
+     , r.data->>'Model'                          as "model"
+     , r.data->>'Vendor Analyte Name'            as "vendor_analyte_name"
+     , r.data->>'Vendor Specimen Description'    as "vendor_specimen_description"
+     , r.data->>'Vendor Result Description'      as "vendor_result_description"
+     , r.data->>'Test Performed LOINC Code'      as "test_performed_loinc_code"
+     , r.data->>'Test Performed LOINC Long Name' as "test_performed_loinc_long_name"
+     , r.data->>'Test Ordered LOINC Code'        as "test_ordered_loinc_code"
+     , r.data->>'Test Ordered LOINC Long Name'   as "test_orderd_loinc_long_name"
+     , r.data->>'Vendor Comment'                 as "vendor_comment"
+     , r.data->>'Vendor Analyte Code'            as "vendor_analyte_code"
+     , r.data->>'Vendor Reference ID'            as "vendor_reference_id"
+     , r.data->>'Testkit Name ID'                as "testkit_name_id"
+     , r.data->>'Testkit Name ID Type'           as "testkit_name_id_type"
+     , r.data->>'Equipment UID'                  as "equipment_uid"
+     , r.data->>'Equipment UID Type'             as "equipment_uid_type"
+     , r.data->>'Component'                      as "component"
+     , r.data->>'Property'                       as "property"
+     , r.data->>'Time'                           as "time"
+     , r.data->>'System'                         as "system"
+     , r.data->>'Scale'                          as "scale"
+     , r.data->>'Method'                         as "method"
+     , r.data->>'Publication Version ID'         as "publication_version_id"
+     , r.data->>'LOINC Version ID'               as "loinc_version_id"
+     -- additional fields we add for our own needs
+     , r.data->>'is_otc'                         as "is_otc"
+     , r.data->>'is_home'                        as "is_home"
+     , r.data->>'is_serial'                      as "is_serial"
+     , r.data->>'is_unproctored'                 as "is_unproctored"
+     , r.data->>'fda_ref'                        as "fda_ref"
+     , r.data->>'fda_authorization'              as "fda_authorization"
+     , r.data->>'processing_mode_code'           as "processing_mode_code"
+FROM
+    lookup_table_row r
+        JOIN lookup_table_version v ON r.lookup_table_version_id = v.lookup_table_version_id
+        and upper(v.table_name) = 'LIVD-SARS-COV-2'
+ORDER BY
+        r.data->>'Manufacturer'
+       , r.data->>'Model'
+       , r.data->>'Equipment UID'
+       , r.row_num
+;
+
+/* Create all our views */
+CREATE INDEX idx_vw_livid_manufacturer on vw_livd_table(manufacturer);
+CREATE INDEX idx_vw_livid_model on vw_livd_table(model);
+CREATE INDEX idx_vw_livid_test_performed_loinc_code on vw_livd_table(test_performed_loinc_code);
+CREATE INDEX idx_vw_livid_test_orderd_loinc_code on vw_livd_table(test_ordered_loinc_code);
```

Comment:
nit: spelling of index name (`orderd` vs `ordered`)

Reply:
I'll fix it.